### PR TITLE
Makes tool act params named consistently.

### PR DESCRIPTION
**Pull request by ShizCalev**
also fixes pipe wrench_act typecasting

### CODE REVIEW

File: code/game/machinery/pipe/construction.dm [hunk 1]
```diff
@@ -125,7 +125,7 @@ Buildable meters
 	setDir(turn(dir,-90))
 	fixdir()
 
-/obj/item/pipe/wrench_act(mob/living/user, obj/item/wrench/W)
+/obj/item/pipe/wrench_act(mob/living/user, obj/item/tool)
```
Comment (ShizCalev) [inline code/game/machinery/pipe/construction.dm]: This seems to have been an oversight/mistake during conversion. wrench_act would only call if the item's tool_type was `TOOL_WRENCH`

**Comment by ACCount12** [comment]
I actually replaced `tool` with `I`, which stands for `obj/[i]tem`. For the sake of consistency with `obj/item/I` in most attackby procs.

**Comment by ShizCalev** [comment]
oh i forgot your PR was still open.

File: code/game/machinery/pipe/construction.dm (hunk 1 above)
Comment (ShizCalev) [reply code/game/machinery/pipe/construction.dm]: @ACCount12 Would you mind rolling this into your PR?

**Pull request closed without merging**